AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.ts

```diff
@@ -50,18 +52,23 @@ export class AddParticipantComponent extends AddParticipantBaseDirective impleme
     @ViewChild(ParticipantListComponent, { static: true })
     participantsListComponent: ParticipantListComponent;
 
-    private judiciaryRoles = Constants.JudiciaryRoles;
+    public judiciaryRoles = Constants.JudiciaryRoles;
 
     constructor(
         private searchService: SearchService,
         protected videoHearingService: VideoHearingsService,
         private participantService: ParticipantService,
         protected router: Router,
         protected bookingService: BookingService,
+        protected featureFlagService: FeatureFlagService,
         protected logger: Logger
     ) {
         super(bookingService, router, videoHearingService, logger);
         this.titleList = searchService.TitleList;
+        featureFlagService
+            .getFeatureFlagByName('EJudFeature')
+            .pipe(first())
+            .subscribe(result => (this.judiciaryRoles = result ? Constants.JudiciaryRoles : []));
```

Comment:
for the if check you want to use triple equals "===" so it should be like this.judiciaryRoles === result

Reply:
here i am not comparing this.judiciary roles and result, i setting value to this.judiciaryRoles